tests/shipment_test.go

```diff
@@ -271,3 +271,27 @@ func (c *ClientTests) TestShipmentLowestRate() {
 	lowestRate, err = client.LowestShipmentRateWithCarrier(shipment, []string{"BAD_CARRIER"})
 	assert.Error(err)
 }
+
+func (c *ClientTests) TestShipmentForm() {
+	client := c.TestClient()
+	assert, require := c.Assert(), c.Require()
+
+	shipment, err := client.CreateShipment(c.fixture.OneCallBuyShipment())
+	require.NoError(err)
+
+	formType := "return_packing_slip"
+
+	// Test shipment form with no additional options
+	shipmentWithForm, err := client.GenerateShipmentForm(shipment.ID, formType)
+	require.NoError(err)
+	assert.Len(shipmentWithForm.Forms, 1)
+	assert.Equal(formType, shipmentWithForm.Forms[0].FormType)
+	assert.NotNil(formType, shipmentWithForm.Forms[0].FormURL)
+
+	// Test shipment form with additional options
+	shipmentWithForm, err = client.GenerateShipmentFormWithOptions(shipment.ID, formType, c.fixture.RmaFormOptions())
+	require.NoError(err)
+	assert.Len(shipmentWithForm.Forms, 1)
+	assert.Equal(formType, shipmentWithForm.Forms[0].FormType)
+	assert.NotNil(formType, shipmentWithForm.Forms[0].FormURL)
```

Comment:
Issue: I'm unsure how this passed because it's not proper syntax
```suggestion
	assert.NotNil(shipmentWithForm.Forms[0].FormURL)
```